src/components/pages/PageCards/PageCards.jsx

```diff
@@ -52,58 +58,51 @@ function PageCards() {
       <CreateCardsBtn>+ Create A Card</CreateCardsBtn>
 
       <div data-test="page-cards" className="row">
-        <If condition={cards.length <= 0}>
+        <If condition={!cards}>
 
-          <div data-test="page-cards__alt-message" className="container">
+          <div data-test="p-cards__alt-message" className="container">
             <h2>You don&apos;t have any cards yet</h2>
             <CreateCardsBtn>Create A Card</CreateCardsBtn>
           </div>
 
           <Else>
+            <h2 className="col-12">Character Cards</h2>
             {cards.map((card, index) => (
-              <div
-                className="col-12 col-sm-5 col-md-4 col-lg-3"
+              <If
+                condition={card.type === 'Character'}
                 key={card.id}
-                data-test="page-cards__card"
               >
+                <CardsMap
+                  card={card}
+                  handleDelete={(e) => handleDelete(e, index)}
+                />
+              </If>
+            ))}
 
-                <h2
-                  data-test="page-cards__type"
-                  className="px-3"
-                >
-                  {card.type}
-                </h2>
-
-                <CardsDisplay card={card} />
-
-                <h3
-                  data-test="page-cards__title"
-                  className="px-3 text-secondary"
-                >
-                  {card.cardTitle}
-                </h3>
-
-                <button
-                  type="button"
-                  className="btn-primary"
-                  onClick={(e) => {
-                    e.preventDefault();
-                    history.push(`/cards-edit/${card.id}`);
-                  }}
-                >
-                  Edit
-                </button>
-
-                <button
-                  data-test="page-cards__delete-card"
-                  className="btn-warning"
-                  type="button"
-                  onClick={(e) => handleDelete(e, index)}
-                >
-                  Delete
-                </button>
-
-              </div>
+            <h2 className="col-12">Circumstance Cards</h2>
```

Comment:
There is a row, I think you just missed it because I had two attributes on one line. 

For col-12 I was getting some weird resizing issues without it, but I'll double check.